src/sceneEventHandler.js

```diff
@@ -149,6 +159,8 @@ window.addEventListener( 'mousedown', () => {
               jsonObj.planets[3].beingViewed = true;
               jsonObj.traversal = true;
 
+              document.getElementById("TextBox").innerHTML = jsonObj.planets[3].facts[ranNum];
```

Comment:
you should store the textbox element as a member somewhere to avoid doing a lookup every click.

Reply:
This has been altered due to the AR implementation